source/Tutorials/Advanced/Simulators/O3DE/Setting-Up-Simulation-O3DE-Advanced.rst

```diff
@@ -0,0 +1,132 @@
+
+
+Prerequisites
+-------------
+Ensure you have the following before starting:
+
+- O3DE set up on your machine. For instructions, follow the `O3DE installation for Ubuntu <Installation-Ubuntu>` guide.
+- ROS 2 (Foxy or later) installed on your system.
+- The ``o3de-extras`` repository cloned locally (on the ``stabilization`` branch).
+- **MoveIt**, which is used for motion planning in ROS 2. Follow the MoveIt 2 documentation for installation instructions.
+
+
+Creating a New ROS 2 Project
+----------------------------
+
+1. **Register the Template**:
+
+Register the template with O3DE so that it can be used to create a new project.
+
+From the root directory of your O3DE installation, run:
+
+.. code-block:: bash
+
+   ./scripts/o3de.sh register --all-templates-path <path_to_o3de_extras>/Templates
+
+This command registers all templates in the ``o3de-extras`` repository, including the Robotic Manipulation Template.
+
+2. **Create a New Project**:
+
+Create a new project using the ROS 2 Robotic Manipulation Template:
+
+.. code-block:: bash
+
+   ./scripts/o3de.sh create-project --project-name <project_name> --template-name Ros2RoboticManipulationTemplate --project-path <path-to-project-directory>
+
+This will generate a new project directory with the necessary files and configurations.
+
+3. **Install Dependencies**:
+
+Navigate to your project directory:
+
+.. code-block:: bash
+
+   cd <project_path>
+
+Install the required Python packages and dependencies for the project. Typically, you will need to install ROS 2 and MoveIt dependencies. This can often be done with:
+
+.. code-block:: bash
+
+   sudo apt install ros-${ROS_DISTRO}-moveit ros-${ROS_DISTRO}-moveit-resources ros-${ROS_DISTRO}-depth-image-proc
+
+Ensure that you also have any additional dependencies specified in the project's ``requirements.txt`` or equivalent configuration files.
+
+4. **Configure and build the Project**:
+
+After installing dependencies, cofigure and build the project using the following commands:
+
+.. code-block:: bash
+
+   cmake -B build/ -S . -G "Ninja Multi-Config"
+
+.. code-block:: bash
+
+   cmake --build <path-to-build-directory> --target <project_name> Editor
+
+Ensure the build completes without errors.   
+
+
+Configurations and launch of the project
+----------------------------------------
+
+1. **Configure ROS 2 and MoveIt**:
+
+The template may include configuration files for ROS 2 and MoveIt. Ensure these are properly configured to match your simulation setup. Key files include:
+
+- **ROS 2 Launch Files**: Typically found in the ``launch`` directory, configure these files to start the ROS 2 nodes required for your simulation.
+- **MoveIt Configuration**: Check the ``moveit_config`` directory for MoveIt configuration files. Ensure these files are correctly set up for your robot and planning requirements.
+
+2. **Launch the Simulation**:
+
+Start the O3DE Editor:
+
+.. code-block:: bash
+
+   <path-to-o3de-directory>/build/bin/profile/Editor
```

Comment:
You might use the game launcher directly, there is no need to go to Editor at this stage.